core/backend/src/Material.ts

```diff
@@ -198,6 +198,10 @@ export class RenderMaterialElement extends DefinitionElement {
       }
       if (!Id64.isValid(sourceMap.TextureId) || sourceMap.TextureId === undefined)
         continue;
+      // textureId has a chance to be a base 10 number instead of a hexadecimal string. This was observed in an iModel created with the Revit Connector.
```

Comment:
This is going to cause problems in places other than `onCloned`.

Reply:
Any issues with mapping textures to invalid in the target instead of trying to recover with a potentially inaccurate base 10 number?

Reply:
I think JS can represent ints from 0..2**53 − 1, so it still covers a lot of ID space accurately. So technically it can be inaccurate, but probably works out in practice "most" of the time. Depends how pedantic you want to be. Since we cannot easily clean up old iModels, that would also compromise visual fidelity for a notable subset.

Reply:
I am not convinced that always returning an invalid Id when we encounter a number is a good idea, vs returning a valid Id. My comment from 3 weeks ago was pointing out that this specific line of code in this specific function is not the only place that "whoops it's a number not a string" will break stuff.